src/iris/api.py

```diff
@@ -1791,6 +1791,13 @@ def on_post(self, req, resp):
                                                    `current_step`, `active`, `application_id`, `bucket_id`)
                            VALUES (:plan_id, :created, :context, 0, :active, :application_id, :bucket_id)''',
                         data).lastrowid
+                    # adding additional context
+                    if self.enable_triage_context or plan_id in self.triage_allow_list:
+                        context['incident_id'] = incident_id
```

Comment:
these iris context variables are not meant to exist in the top level of the context but rather inside the 'iris' key within the context i.e. context['iris']['incident_id']

Reply:
added